backend/controllers/globalSearch.js

```diff
@@ -3,7 +3,7 @@ const shortid = require('shortid')
 const log = require('../log')
 const { isPrisonerIdentifier } = require('../utils')
 
-const globalSearchFactory = (elite2Api, globalSearchApi) => {
+const globalSearchFactory = globalSearchApi => {
   const searchByOffender = (context, offenderNo, gender, location, dateOfBirth) =>
     // offenderNo can be removed once we have fully switched to prisoner offender search instead, i.e. the env variable
     // OFFENDER_SEARCH_API_ENABLED has been removed
```

Comment:
Has this comment been addressed?

Reply:
wanted to get the change working before addressing comment - will pick up next - thanks